spring/boot2-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaSslConfigurationTest.java

```diff
@@ -64,15 +61,6 @@
 
     @SpringBootApplication
     public static class TestConfiguration {
-
-        @Bean
-        public ArmeriaServerConfigurator okService() {
```

Comment:
Already has a ArmeriaOkServiceConfiguration in same package.

Reply:
Then could you remove the `OkService` underneath, please?